runtime/queries/nix/injections.scm

```diff
@@ -0,0 +1,45 @@
+; mark arbitary languages with a comment
+(
+  ((comment) @injection.language) . [
+    (indented_string_expression (_) @injection.content)
+    (_ (_)* (indented_string_expression (_) @injection.content))
+    (_ (_)* (_ (_)* (indented_string_expression (_) @injection.content)))
+    (_ (_)* (_ (_)* (_ (_)* (indented_string_expression (_) @injection.content))))
+    (_ (_)* (_ (_)* (_ (_)* (_ (_)* (indented_string_expression (_) @injection.content)))))])
```

Comment:
yeah I noticed this as well, I suppose being more explicit is good in this case, perhaps we should add a comment _after_ the string as well, for where it might look better stylistically.

Reply:
done